src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,22 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany(fetch = FetchType.EAGER)
```

Comment:
in my solution, Maks left a comment about this relation - does one smile can be used in many comments?

Reply:
Tricky question since maybe each smile doesn't need to know every comment where it was used...

Reply:
still, can be?) for me it would've been very convenient, one time, search for every comment that used specific smile. for statistic

Reply:
ok) fixed